makepkg.sh

```diff
@@ -45,7 +45,7 @@ declare -r startdir="$PWD"
 
 packaging_options=('strip')
 other_options=('ccache' 'distcc' 'buildflags' 'makeflags')
-splitpkg_overrides=('pkgdesc' 'arch' 'url' 'license' 'groups' 'depends'
+splitpkg_overrides=('pkgdesc' 'arch' 'url' 'license' 'groups' 'backup' 'depends'
```

Comment:
when I looked at the summary, my first thought was "what does arch do for their config files?"  now I see that 'backup' is in this list twice